resources/assets/js/app/views/comments/TopicCommentNew.js

```diff
@@ -0,0 +1,210 @@
+var _ = require('underscore');
+var Radio = require('backbone.radio');
+var Dropzone = require('dropzone');
+var currentUser = require('../../initializers/currentUser');
+var AttachmentModel = require('../../models/AttachmentModel');
+var App = require('../../instances/appInstance');
+
+module.exports = Marionette.ItemView.extend({
+    template: 'TopicCommentNew',
+    _dropZone: null,
+    _files: [],
+
+    ui: {
+        'submit': '#submit',
+        'close': '.close',
+        'errors': '.errors',
+        'loader': '.loader',
+        'commentDlg': '#commentdlg'
+    },
+
+    modelEvents: {
+        'invalid': function (model, errors, options) {
+            //logger('model is invalid', errors, options);
+            this.showLoader(false);
+            var ui = this.ui;
+            ui.errors.empty();
+            errors.message.forEach(function (message, i) {
+                ui.errors.append(message);
+            });
+            this.showErrors(true);
+        },
+    },
+
+    events: {
+        'click @ui.submit': 'submitComment',
+    },
+
+    showLoader: function (show) {
+        return this.ui.loader.toggleClass('hidden', !show);
+    },
+
+    showErrors: function (show) {
+        return this.ui.errors.toggleClass('hidden', !show);
+    },
+
+    submitComment: function (event) {
+        event.preventDefault();
+
+        this.showErrors(false);
+        this.showLoader(true);
+
+        var user_id = currentUser.toJSON().id;
+
+        var data = {
+            user_id: user_id
+        };
+
+        _.each(this.$('form').serializeArray(), function(input) {
+            data[ input.name ] = input.value;
+        });
+
+        var view = this;
+
+        this.model.save(data, {
+            success: function (model) {
+                if (view._dropZone && view._dropZone.files.length) {
+                    // start upload to server
+                    view._dropZone.processQueue();
+                }
+                Radio.channel('сommentCollection').trigger('addComment', model);
+                view.ui.commentDlg.modal('hide');
+            },
+
+            error: function (model, response) {
+                view.showErrors(true);
+                view.showLoader(false);
+                view.ui.errors.empty().append(response.responseText);
+            },
+        });
+    },
+
+    initDropZone: function () {
+        var view = this;
+        var attachModel = new AttachmentModel();
+        this._dropZone = new Dropzone(this.$('#drop')[0], {
+            url: function(file) {
+                return App.getBaseUrl() + _.result(view.model, 'url') + _.result(attachModel, 'url');
+            },
+            method: 'post',
+            // input file name, registered on server
+            paramName: "f",
+            parallelUploads : 10,
+            hiddenInputContainer: '#drop',
+            autoProcessQueue : false,
+            maxFilesize: 8,
+            maxFiles: 5,
```

Comment:
I think that parameters like:
- parallelUploads
- maxFilesize
- maxFiles 
should be moved to config file